predicators/ground_truth_models/kitchen/operators.py

```diff
@@ -0,0 +1,66 @@
+"""Ground-truth Operators for the Kitchen environment."""
+
+from typing import Dict, Set
+
+from predicators.structs import LiftedAtom, Predicate, STRIPSOperator, Type, \
+    Variable
+
+
+class KitchenGroundTruthOperatorFactory():
+    """Ground-truth Operators for the Kitchen environment."""
+
+    @staticmethod
+    def get_operators(env_name: str, types: Dict[str, Type],
+                      predicates: Dict[str, Predicate]) -> Set[STRIPSOperator]:
+        """Creates Operators for Mujoco Kitchen Env."""
+        assert env_name == "kitchen"
+        # Types
+        gripper_type = types["gripper"]
+        object_type = types["obj"]
+
+        # Objects
+        gripper = Variable("?gripper", gripper_type)
+        obj = Variable("?obj", object_type)
+        obj2 = Variable("?obj2", object_type)
+
+        # Predicates
+        At = predicates["At"]
+        TurnedOn = predicates["TurnedOn"]
+        OnTop = predicates["OnTop"]
+
+        operators = set()
+
+        # MoveTo
+        parameters = [gripper, obj]
+        preconditions: Set[LiftedAtom] = set()
+        add_effects = {LiftedAtom(At, [gripper, obj])}
+        delete_effects: Set[LiftedAtom] = set()
+        move_to_nsrt = STRIPSOperator("MoveTo", parameters, preconditions,
```

Comment:
Nit: can you change the name of this and to `move_to_operator` instead of `move_to_nsrt`? Just don't want to confuse ourselves in the future